org-journal.el

```diff
@@ -743,6 +751,14 @@ buffer not open already, otherwise `nil'.")
           (kill-region start end)))
       (string-empty-p (org-trim (buffer-string))))))
 
+(defun org-journal-remove-drawer ()
+  "Removes the drawer configured via org-journal-skip-carryover-drawers"
+  (interactive)
+  (save-excursion
+    (goto-char (point-min))
```

Comment:
@cslux That's a good point. This might break other workflows having same file for the journal (if they have configured the variable). I haven't looked much deeply, but is there any elisp function to go to the beginning of the journal's current entry ?

Reply:
I think this should do the trick.
``` emacs-lisp
(save-excursion
 (save-restriction
   (unless (org-journal-daily-p) 
     (while (org-up-heading-safe))
     (org-narrow-to-subtree))
    (goto-char (point-min))
    ...
))
```

Reply:
Thanks, that did the trick!